molecule/builder/Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM ubuntu:14.04
+# ubuntu:14.04 as of 2017-12-15
+FROM ubuntu@sha256:084989eb923bd86dbf7e706d464cf3587274a826b484f75b69468c19f8ae354c
```

Comment:
I don't like embedding the hash right into the Dockerfile. It's more annoying to over-ride programtically in the future if need be. I got a couple commits coming in with some slightly tweaked logic.

Reply:
cool. Note that it's also used in other Docker files so maybe you'd like to fix in a uniform way.